.gitignore

```diff
@@ -1,4 +1,5 @@
 /dist/
 /*.egg-info/
 /.coverage
+/COLCON_IGNORE
```

Comment:
Confirming that it's intentional for this to only ignore the marker file at the repository root and not in any other places (like test fixture packages). Otherwise the slash should be omitted to allow the pattern to match COLCON_IGNORE files in subdirectories.